config/locales/devise/es.yml

```diff
@@ -116,7 +116,7 @@ es:
         porque ha ingresado incorrectamente la clave personal demasiadas veces.
       otp_delivery_preference:
         instruction: Puede cambiar su elección la próxima vez que inicie una sesión.
-        phone_unsupported: NOT TRANSLATED YET
+        phone_unsupported: NOT TRANSLATED YET %{location}
```

Comment:
There is a weird case where a translation is missing interpolation args b/c it hasn't been translated yet. I worked around that by simply appending the interpolation arg to the end of the `NOT TRANSLATED YET` message.